src/url-handling.js

```diff
@@ -223,25 +223,6 @@ export function stateFromLocation(location: Location): UrlState {
   };
 }
 
-function toValidTabSlug(slug: ?string): TabSlug {
-  const defaultTab = 'calltree';
-  switch (slug) {
-    case 'calltree':
-    case 'stack-chart':
-    case 'marker-chart':
-    case 'marker-table':
-      return slug;
-    case undefined:
-      return defaultTab;
-    default:
-      console.error(
-        'Unknown tab found, maybe a URL upgrader needs to be written.',
-        slug
-      );
```

Comment:
I think we miss this error now. But I think the Flow check brings a better protection.